### ISSUE TITLE
Chrome not launching on Mac with 1.7.3, works with 1.7.1

### ISSUE THREAD

**Issue by nruth** [testem/testem#880]
Hi

Using testem through ember-cli 2.5.0, `ember test --serve` only runs phantomjs. Firefox and Chrome fail to launch or are skipped somehow.

I'm on Mac OS 10.11.5.

My testem file looks like this:

```
/*jshint node:true*/
module.exports = {
  "framework": "qunit",
  "test_page": "tests/index.html?hidepassed",
  "ignore_missing_launchers": true,
  "disable_watching": true,
  "launch_in_ci": [
    "PhantomJS",
    "Chrome",
    "Firefox",
    "Opera",
    "Safari"
  ],
  "launch_in_dev": [
    "PhantomJS",
    "Chrome",
    "Firefox"
  ],
  "phantomjs_debug_port": 9000
};
```

No errors, it just ignores the launchers. I've tried removing "ignore_missing_launches" but the same thing happens.

![screen shot 2016-05-27 at 17 28 49](https://cloud.githubusercontent.com/assets/26158/15613386/8dc19b44-2433-11e6-9d43-da7111d5308e.png)

After doing `npm install --save-dev testem@1.7.1` it starts working again.

![screen shot 2016-05-27 at 17 53 14](https://cloud.githubusercontent.com/assets/26158/15613476/eeff6990-2433-11e6-895f-bd49c30f0bd2.png)

1.7.2 hangs at "Waiting for runners..." but I guess that's known about since the release was deprecated?

![screen shot 2016-05-27 at 18 00 44](https://cloud.githubusercontent.com/assets/26158/15613683/fff9507a-2434-11e6-86e5-4ddaba10ee43.png)

Could this be a regression from 22fc6afc328888344b22d18c754b0c5b6eb5d289 or did an API change?

**Comment by johanneswuerbach** [comment]
Thanks for the detailed report, fixed will be release once CI is green: https://github.com/testem/testem/pull/881.

**Comment by nruth** [comment]
Great :) Thanks for the super-quick fix.

**Comment by johanneswuerbach** [comment]
Released as `v1.7.4`
